testing/library/values_validation/values_validation.go

```diff
@@ -109,7 +109,13 @@ func ValidateJSONValues(validator *validation.ValuesValidator, moduleName string
 	}
 
 	valuesKey := utils.ModuleNameToValuesKey(moduleName)
-	err = validator.ValidateModuleValues(valuesKey, obj)
+
+	if configValues {
+		err = validator.ValidateModuleConfigValues("config", obj)
+	} else {
+		err = validator.ValidateModuleValues(valuesKey, obj)
+	}
+
```

Comment:
Can you discussed this code with @diafour ?

Reply:
Discussed with @nabokihms.